### ISSUE TITLE
Clarify iocage fetch output

### ISSUE THREAD

**Issue by Mrt134** [iocage/iocage#228]
Version 0.9.8.1 05/23/2017 
Copy/paste:

```
[root@freenas ~]# iocage fetch                                                  
[0] 9.3-RELEASE (EOL)                                                           
[1] 10.1-RELEASE (EOL)                                                          
[2] 10.2-RELEASE (EOL)                                                          
[3] 10.3-RELEASE                                                                
[4] 11.0-RELEASE                                                                
                                                                                
Which release do you want to fetch? (11.0)                                      
Type EXIT to quit: 
```

The current options are ambiguous, as the "(11.0)" makes it look like the 11.0-RELEASE is already
 selected to be fetched, with the only required user input being to type EXIT to continue. Suggested
 text rework:
"Type the number of the release to fetch, or type EXIT to quit without (downloading/fetching):"

Make sure to follow and check these boxes before submitting an issue! Thank you.

- [x] Supply `iocage --version`
- [x] Supply the commands used, along with any steps to recreate it.
- [x] Provide the output from the command you issued.
- [x] Supply what you expected the result or output to be
- [x] Checked that the problem has not already been fixed on `master` if using
 a stable release.

**Comment by skarekrow** [comment]
This actually works as expected on a RELEASE:
```
╰─ iocage fetch
[0] 9.3-RELEASE (EOL)
[1] 10.1-RELEASE (EOL)
[2] 10.2-RELEASE (EOL)
[3] 10.3-RELEASE
[4] 11.0-RELEASE

Which release do you want to fetch? (11.0-RELEASE)
Type EXIT to quit:
Fetching: 11.0-RELEASE

^C
Aborted!
```
In this example I only hit enter, which uses the expected entry in the parenthesis, IE: `11.0-RELEASE`. This bug would seem to stem from the fact you're using it on FreeNAS and it becomes `11.0` instead, which isn't a valid entry. I'm not crazy about reworking the text to the suggested one as it creates a long string before the user gets to type. On a 80 character terminal, I could see the prompt becoming annoying.

I'm all ears on any alternative text besides the one suggested though, as I suppose it could be clearer.

**Comment by skarekrow** [comment]
So the bug that was found in your paste was fixed with fd1139087b672152f3bab1252904df1fd934d7c7, I'm leaving this enhancement open so we can workshop better fetching text.

**Comment by Mrt134** [comment]
How about a few lines outlining the options? 
Rough example (exact words aren't important):
```
 iocage fetch
[0] 9.3-RELEASE (EOL)
[1] 10.1-RELEASE (EOL)
[2] 10.2-RELEASE (EOL)
[3] 10.3-RELEASE
[4] 11.0-RELEASE

Type the number of the desired release, OR
Press [enter] to fetch the default option: (11.0-RELEASE), OR
Type EXIT to quit:
```
